environment.yml

```diff
@@ -0,0 +1,5 @@
+name: gpm_api
+channels:
+  - conda-forge
+dependencies:
+  - cartopy>=0.20.0
```

Comment:
You don't want to install also the other dependencies via micromamba?

Reply:
Some packages are not available in `conda-forge` (h5py, pyresamble, wget, and ximage), so I followed what's in the installation doc

Reply:
https://anaconda.org/conda-forge/h5py, https://anaconda.org/conda-forge/wget, https://anaconda.org/conda-forge/pyresample in case ;)

Reply:
My bad, I had typos in my environment.yml when trying to install those...